packages/muelu/src/MueCentral/MueLu_VerbosityLevel.hpp

```diff
@@ -78,6 +78,8 @@ namespace MueLu {
       External        = 0x01000000, //!< Print external lib objects
       Debug           = 0x02000000, //!< Print additional debugging information
 
+      Developer       = 0x03000000, //!< Print information primarily of interest to developers
```

Comment:
Is the intent to have `Developer = External+Debug`?

Reply:
No, I just don't read hex :(.   Lemme fix this.